content/tutorial/14-converters.md

```diff
@@ -47,18 +47,16 @@ There are multiple errors that this command may raise which you'll want to handl
 
 This command quite simply takes a member and a reason and sends them a message. Note here that we can just immediately access the member's `send` method, since the converter guarantees we either get a `Member` object, or an exception is raised if conversion fails. In this command, `member` will **never** be an object of any type other than `Member`.
 
-Try making a few of your own commands use converters, if you don't already! A few converters of interest to you might be:
-- `discord.TextChannel`
-- `discord.Colour`
-- `discord.Role`
+Try making a few of your commands use converters, if you don't already! A few converters of interest to you might be:
+- [`discord.TextChannel`](https://discordpy.readthedocs.io/en/latest/api.html?highlight=textchannel#discord.TextChannel)
+- [`discord.Colour`](https://discordpy.readthedocs.io/en/latest/api.html?highlight=colour#discord.Colour)
+- [`discord.Role`](https://discordpy.readthedocs.io/en/latest/api.html?highlight=role#discord.Role)
 
 ## Creating a Custom Converter
 
 So, we've covered the stuff discord.py provides by default, but what if you want more than what it gives? Well, fortunately for us discord provides a base converter class which we can subclass to make our own: `discord.ext.commands.Converter`
 
-{{< tip "info" >}}
-For the sake of code readability and reuse, I suggest when making converters you put them in their own file and allow them to be imported into relevant cogs.
-{{< /tip >}}
+{{< tip "info" >}} For the sake of code readability and reuse, I suggest when making converters you put them in their file and allow them to be imported into relevant cogs. {{</tip >}}
```

Comment:
This shouldn't have been inlined, I find it easier to work with when they're on separate lines (also, bear in mind that while this means the same to Hugo, it may not mean the same to my own utilities I occasionally use to do things)